Neos.Media/Classes/Domain/Model/Asset.php

```diff
@@ -412,6 +445,56 @@ public function setAssetCollections(Collection $assetCollections)
         $this->assetCollections = $assetCollections;
     }
 
+
+    /**
+     * @param string $assetSourceIdentifier
+     */
+    public function setAssetSourceIdentifier(string $assetSourceIdentifier): void
```

Comment:
Should this actually be settable? What happens if I set this to something else?

Reply:
Yes, needs to be settable, but I'll document it as internal-use only.